content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -112,11 +112,16 @@ documentation for the plugins about what port(s) those need.
 
 ## Installing runtime {#installing-runtime}
 
-Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+To run containers in Pods, Kubernetes uses a
+{{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
 
-Since v1.14.0, kubeadm will try to automatically detect the container runtime on Linux nodes
+By default, Kubernetes uses the
+{{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
+to interface with your chosen container runtime.
+
+kubeadm automatically tries to detect the container runtime on Linux nodes
 by scanning through a list of well known domain sockets. The detectable runtimes and the
-socket paths, that are used, can be found in the table below.
+socket paths that are tried are listed in the following table:
```

Comment:
@sftim what do you think about:
line 122: Kubeadm instead of kubeadm (start of sentence)
line 123: You can find the container runtimes and associated socket paths in the following table:

Reply:
Tweaked to avoid the question.